source/views/home/edit.js

```diff
@@ -235,18 +239,39 @@ class EditHomeView extends React.PureComponent<void, Props, State> {
     this.setState(() => ({width: event.window.width}))
   }
 
+  onRowActive = event => {
+    this.setState(() => ({activeRowView: event.rowData.index}))
+  }
+
+  onRowMoved = event => {
+    let order = [...this.props.order]
+    order.splice(event.to, 0, order.splice(event.from, 1)[0])
+    this.props.onSaveOrder(order)
+  }
+
+  onMoveEnd = () => {
+    this.setState(() => ({activeRowView: ''}))
```

Comment:
If this line is moved to `onRowMoved`, there's a chance that a single selection will leave that row with `active` styles. It behaves appropriately in `onMoveEnd` as that does not care if the row was moved or not.